products/rtpengine.md

```diff
@@ -0,0 +1,48 @@
+---
+title: rtpengine 
+category: server-app
+permalink: /rtpengine
+versionCommand: rtpengine -v
+releaseDateColumn: true
+
+identifiers:
+-   repology: rtpengine
+auto:
+  methods:
+    - git: https://github.com/sipwise/rtpengine.git
+      regex: ^mr(?P<major>\d+)_(?P<minor>\d+)_(?P<patch>\d{1,3})_?(?P<tiny>\d+)?$
+
+releases:
+
+-   releaseCycle: "mr13.0"
+    releaseDate: 2024-10-01
+    eol: true
```

Comment:
Could you document the last supported release (13.1 at the time of writing) ?

Reply:
done.